silkworm/rpc/http/connection.hpp

```diff
@@ -65,6 +65,7 @@ class Connection : public StreamWriter {
     /* StreamWriter Interface */
     Task<void> open_stream() override;
     Task<void> close_stream() override;
+    size_t get_capacity() override { return 4 * 1024; }
```

Comment:
Better use a constant instead of an hard-coded magic number

Reply:
ok